src/frontend/src/pages/CreateWorkPackagePage/CreateWPFormView.tsx

```diff
@@ -7,171 +7,235 @@ import Box from '@mui/material/Box';
 import Grid from '@mui/material/Grid';
 import Button from '@mui/material/Button';
 import TextField from '@mui/material/TextField';
-import Typography from '@mui/material/Typography';
 import InputAdornment from '@mui/material/InputAdornment';
 import { DatePicker } from '@mui/x-date-pickers/DatePicker';
 import { routes } from '../../utils/routes';
-import { EditableTextInputListUtils, FormStates } from './CreateWPForm';
-import EditableTextInputList from '../../components/EditableTextInputList';
 import PageTitle from '../../layouts/PageTitle/PageTitle';
 import PageBlock from '../../layouts/PageBlock';
+import * as yup from 'yup';
+import { Controller, useFieldArray, useForm } from 'react-hook-form';
+import { yupResolver } from '@hookform/resolvers/yup';
+import { useQuery } from '../../hooks/utils.hooks';
+import ReactHookTextField from '../../components/ReactHookTextField';
+import { FormControl, FormLabel, IconButton } from '@mui/material';
+import ReactHookEditableList from '../../components/ReactHookEditableList';
+import DeleteIcon from '@mui/icons-material/Delete';
+import { bulletsToObject } from '../../utils/form';
+import { wbsPipe } from '../../utils/pipes';
+import { SubmitButton } from '../../components/SubmitButton';
+import { wbsTester } from '../CreateChangeRequestPage/CreateChangeRequestView';
+
+const schema = yup.object().shape({
+  name: yup.string().required('Name is required'),
+  wbsNum: yup.string().required('WBS Number is required').test('wbs-num-valid', 'WBS Number is not valid', wbsTester),
+  crId: yup
+    .number()
+    .typeError('CR ID must be a number')
+    .required('CR ID is required')
+    .integer('CR ID must be an integer')
+    .min(1, 'CR ID must be greater than or equal to 1'),
+  startDate: yup.date().required('Start Date is required'),
+  duration: yup
+    .number()
+    .typeError('Duration must be a number')
+    .required('Duration is required')
+    .integer('Duration must be an integer')
+    .min(1, 'Duration must be greater than or equal to 1')
+});
 
 interface CreateWPFormViewProps {
-  states: FormStates;
-  dependencies: string[];
-  initialValues: { name: string; wbsNum: string; crId: string; duration: number };
-  depUtils: EditableTextInputListUtils;
-  expectedActivities: string[];
-  eaUtils: EditableTextInputListUtils;
-  deliverables: string[];
-  delUtils: EditableTextInputListUtils;
   allowSubmit: boolean;
-  onSubmit: (e: any) => void;
+  onSubmit: (data: any) => void;
   onCancel: (e: any) => void;
-  startDate: Date | undefined;
 }
 
-const CreateWPFormView: React.FC<CreateWPFormViewProps> = ({
-  states,
-  dependencies,
-  depUtils,
-  expectedActivities,
-  eaUtils,
-  deliverables,
-  delUtils,
-  allowSubmit,
-  onSubmit,
-  onCancel,
-  initialValues,
-  startDate
-}) => {
-  const { name, wbsNum, crId, startDate: setStartDate, duration } = states;
+const CreateWPFormView: React.FC<CreateWPFormViewProps> = ({ allowSubmit, onSubmit, onCancel }) => {
```

Comment:
the grid doesn't align well when it's small here. everything should have `xs={12}` so that it's on its own line when it's super small